app/workers/facilities/dental_service_reload_job.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+# require 'facilities/bulk_json_client'
+require 'common/exceptions'
+# require 'facility_access'
+require 'sentry_logging'
+
+module Facilities
+  class DentalServiceError < StandardError
+  end
+
+  class DentalServiceReloadJob
+    include Sidekiq::Worker
+    include SentryLogging
+
+    def fetch_dental_service_data(file)
+      CSV.read(file, headers: true)
+    rescue => e
+      raise DentalServiceError, "Failed to read CSV file: #{file} - caused by: #{e.cause}"
+    end
+
+    def update_cache(model, facilities)
+      facilities.each do |k|
+        attrs = { station_number: k,
+                  local_updated: Time.now.utc.iso8601 }
+        obj = model.find(k)
+        if obj
+          obj.update(attrs)
+        else
+          model.create(attrs)
+        end
+      end
+
+      logger.info "Updated facility dental service cache for #{facilities.size} facilities"
```

Comment:
Are these log messages necessary in production?

Reply:
Nah. I'll take them out

Reply:
Maybe leave in but change to `debug`?

Reply:
Looks like production level is `info` based on `production.rb` though I'm not sure what else happens in Jenkins. In any case, debug works ok if its necessary